MoonDAO/docs/About/FAQ.md

```diff
@@ -3,32 +3,32 @@ tags: docs/faq, docs/onboarding
 
 ### Why do I need to lock to vote?
-We don’t want our community to be controlled by day-traders. We’d rather be governed by those with a long-term view and commitment to our mission. Locking measures commitment by restricting tokens for some amount of time. For most members the amount and duration is determined when selecting a tier during the onboarding process. Advanced users can purchase any amount of [$MOONEY via Uniswap](https://app.uniswap.org/swap?inputCurrency=ETH&outputCurrency=0x20d4DB1946859E2Adb0e5ACC2eac58047aD41395&chain=mainnet) and [lock](https://www.moondao.com/lock) some or all of that for any amount of time between one week and four years. 
+We want our community to be controlled by those with a long-term view and commitment to our mission. Locking measures commitment by restricting tokens for some amount of time. The amount and duration is determined when selecting during the locking process. Users can purchase any amount of [$MOONEY via Uniswap](https://app.uniswap.org/swap?inputCurrency=ETH&outputCurrency=0x20d4DB1946859E2Adb0e5ACC2eac58047aD41395&chain=mainnet) and [lock](https://www.moondao.com/lock) some or all of that for any amount of time between one week and four years. 
 
 ### What do I get in exchange for locking $MOONEY?
-Locking gives you governance inside of the community. [Our treasury](https://dashboard.moondao.com/#/treasury) is co-governed by all $vMOONEY holders. All our financial decisions are approved by them. You can vote on proposals [here](http://vote.moondao.com/). Those that have 250,000+ voting power can [submit proposals](https://discord.com/channels/914720248140279868/1034923662442254356) directly to the community, otherwise you’ll need a co-signer with 250,000+ voting power.
+Locking gives you governance inside of the community. [Our treasury](https://moondao.com/analytics) is co-governed by all $vMOONEY holders. All our financial decisions are approved by them. You can vote on proposals [here](http://moondao.com/vote).
 
 ### How do I contribute to MoonDAO?
-MoonDAO is organized into teams called “Projects,” you can view the projects that have open roles or are looking for contributors by joining the MoonDAO Discord and visiting the [open-job-board](https://discord.com/channels/914720248140279868/1176926263458406410) channel. Individual projects also have project leads, called Rocketeers. 
+MoonDAO is organized into teams called “Projects,” you can view the ongoing projects that have open roles or are looking for contributors by becoming a Citizen and visiting the [job board](https://moondao.com/jobs), which also includes space job postings from outside Teams.
```

Comment:
This should link to the new projects page where we list currently ongoing projects